packages/react/src/components/DropdownMenu/DropdownMenuItem/DropdownMenuItem.tsx

```diff
@@ -7,10 +7,13 @@ import { DropdownMenuContext } from '../DropdownMenu';
 
 import classes from './DropdownMenuItem.module.css';
 
-export type DropdownMenuItemProps = Omit<
-  ButtonProps,
-  'variant' | 'size' | 'color' | 'fullWidth'
->;
+export type DropdownMenuItemProps = {
+  /**
+   * Change the default rendered element for the one passed as a child, merging their props and behavior.
+   * @default false
+   */
+  asChild?: boolean;
+} & Omit<ButtonProps, 'variant' | 'size' | 'color' | 'fullWidth'>;
```

Comment:
`ButtonProps` already has `asChild`, did it not work correctly before?

Reply:
This is an oversight on my part, good catch! - Will update